instrumentation/grizzly-2.0/src/test/groovy/GrizzlyTest.groovy

```diff
@@ -14,16 +14,18 @@
  * limitations under the License.
  */
 
+
 import io.opentelemetry.auto.test.base.HttpServerTest
+import org.glassfish.grizzly.http.server.HttpServer
```

Comment:
Is this the right order?

Reply:
yeah, at least currently, see [CONTRIBUTING.md](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/CONTRIBUTING.md#intellij-idea):

> With java use the following import layout <b>(groovy should still use the default)</b>

the problem is we don't have any automated formatting / enforcement for groovy imports, so we end up with unnecessary drift sometimes

Reply:
looks like you already decided against it, but you can look into enabling it here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/gradle/spotless.gradle#L9

Reply:
just tried enabling it, but ran into lots of pain, looks like u decided against it too? https://github.com/DataDog/dd-trace-java/blob/master/gradle/spotless.gradle#L19-L26

Reply:
I didn't apply it because I couldn't find a configuration to match the existing files close enough and didn't want significant thrash in the initial PR.

Reply:
i noticed in your commented out config that you skipped all the tests (and I agree, as it mangles the spock data-driven sections). not sure if it's worth applying if we can't apply it to the tests, since those are the majority of our groovy code